Scheduler/feature_scheduler/maintel/fbs_config_sit_survey_block_t248.py

```diff
@@ -0,0 +1,190 @@
+    flush_pad : `float`
+        How long to hold observations in the queue after they
+        were expected to be completed (minutes).
+    """
+    # Define the extra parameters that are used in the greedy survey. I
+    # think these are fairly set, so no need to promote to utility func kwargs
+
+    survey_detailers = [
+        detailers.TrackingInfoDetailer(science_program=name),
+        detailers.CameraRotDetailer(
+            min_rot=np.min(camera_rot_limits), max_rot=np.max(camera_rot_limits)
+        ),
+    ]
+
+    for filtername in sequence:
+        bfs = [
+            (
+                bf.SlewtimeBasisFunction(filtername=filtername, nside=nside),
+                slewtime_weight,
+            ),
+            (bf.StrictFilterBasisFunction(filtername=filtername), stayfilter_weight),
+            (
+                bf.AltAzShadowMaskBasisFunction(
+                    nside=nside,
+                    shadow_minutes=shadow_minutes,
+                    max_alt=max_alt,
+                    min_alt=30.0,
+                ),
+                0,
+            ),
+            (bf.FilterLoadedBasisFunction(filternames=filtername), 0),
+        ]
+
+        exptimes = {filter: exptime for filter in sequence}
+        nexps = {filter: nexp for filter in sequence}
+        nvisits = {filter: visits for filter in sequence}
+        basis_functions = [val[0] for val in bfs]
+        survey = FieldSurvey(
+            basis_functions,
+            ra,
+            dec,
+            sequence=sequence,
+            exptimes=exptimes,
+            nexps=nexps,
+            nvisits=nvisits,
+            nside=nside,
+            ignore_obs=ignore_obs,
+            detailers=survey_detailers,
+            survey_name=name,
+        )
+
+    return survey
+
+
+if __name__ == "config":
+    nside = 32
+    # per_night = True  # Dither DDF per night
+
+    # detailer = detailers.DitherDetailer(max_dither=0.2, per_night=False)
+
+    # n_offsets = 30
+    # night = 42
+    # offsets = np.degrees(detailer._generate_offsets(n_offsets, night))
+
+    camera_ddf_rot_limit = 75.0
+
+    observatory = ModelObservatory(nside=nside, mjd_start=MJD_START)
+    observatory.sky_model.load_length = 3
+    conditions = observatory.return_conditions()
+
+    sky = SkyAreaGenerator(nside=nside)
+    footprints_hp, footprints_labels = sky.return_maps()
+
+    footprints = Footprint(
+        MJD_START,
+        sun_ra_start=conditions.sun_ra,
+        nside=nside,
+        filters=["u", "g", "r_03", "i_06", "z", "y"],
+    )
+    for i, key in enumerate(footprints_hp.dtype.names):
+        footprints.footprints[i, :] = footprints_hp[key]
+    footprints.filters = dict(
+        u=0,
+        g=1,
+        r_03=2,
+        i_06=3,
+        z=4,
+        y=5,
+    )
+
+    filters_list = ["i_06", "r_03"]
+
+    surveys = []
+    i = 0
+    for offset in range(0, 30):
+        surveys.append(gen_field_survey(f"BLOCK-T248_{i}", sequence=filters_list))
```

Comment:
How do you know that the block surveys will be chosen in sequential order? (which seems pretty important, since the dither offsets require that they be chosen in sequential order)

Reply:
Good point. Sequential order was not what I was going for. What this revealed however is that I need to add a relative=False command when offsetting the telescope. Thanks.

Reply:
So basically it's just that every dither position is more than X (how many deg?) from the central point, so that dithers are not "small"? I had somehow thought it was that sequential offsets were large, but that individual pointings may have been close to each other. (kind of like going back and forth across the center of the pointings, so that the telescope moved far enough each time, but that the individual images would be at nearby pointings, just  not consecutively).